src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -291,6 +291,20 @@ type LogoutCommand = {
 
 // Build Image Command Types
 
+/**
+ * Target platform for the image build
+ */
+export type ContainerPlatform = string | {
```

Comment:
@bwateratmsft should the client only accept the { os, architecture } object for platform and require the task to normalize it before calling build? There's pros and cons to both approaches and I'm torn on which I prefer.

Reply:
I think we should only accept one or the other, not both, but I'm also torn on which.

Reply:
let's talk about this at stand up today!